src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC, useState } from 'react';
+import { NavbarBrand } from './NavbarBrand';
+import { NavbarCollapse } from './NavbarCollapse';
+import { NavbarContext } from './NavbarContext';
+import { NavbarToggle } from './NavbarToggle';
+
+export type NavbarCompnentProps = {
+  open?: boolean;
+};
+
+const NavbarCompnent: FC<NavbarCompnentProps> = ({ children, open }) => {
+  const [isOpen, setIsOpen] = useState(open);
+
+  return (
+    <NavbarContext.Provider value={{ isOpen, setIsOpen }}>
+      <nav className="rounded border-gray-200 bg-white px-2 py-2.5 dark:bg-gray-800 sm:px-4">
```

Comment:
Can we add a property to set the navbar as rounded or not?

Reply:
and I think it is better for the navbar to have some border, just like we have in the flowbite documentation site